### PR TITLE
create the order page

### PR DESCRIPTION
**Pull request by AbdullahLaham**
relate #13"

### CODE REVIEW

File: yourOrders/order.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+       <footer>
+        <section class="total">
+            <small>Total</small>
+            <p>$ 29.96</p>
+        </section>
+        <section class="checkout">
+            <button class="plus">
+                +
+            </button>
+            <button class="check">
```
Comment (yaqootturman) [inline yourOrders/order.html:98]: The button is not as design, font color should be white and box color should be black

File: yourOrders/order.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                </div>
+                <div class="counter">
+                    <small>+</small>
+                    <small>1</small>
+                    <small>-</small>
+                </div>
+            </div>
+        </section>
+       </main>
+       <footer>
```
Comment (yaqootturman) [inline yourOrders/order.html:89]: The total part is not footer,  and you need to add a horizontal line as design shows

File: yourOrders/order.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                    <img src="../Images/coffe3.png" alt="coffe image"/>
+                </div>
+                <div class="discription">
+                    <p>Peppermint Mocha</p>
+                    <p>Real chocolate and mint</p>
+                    <small class="grand">GRANDE</small>
+                    <h4>$ 18.98</h4>
+                </div>
+                <div class="counter">
+                    <small>+</small>
```
Comment (yaqootturman) [inline yourOrders/order.html:82]: For all + and - signs need to add circle around them as design shows

File: yourOrders/order.html [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+                </div>
+            </div>
+        </section>
+       </main>
+       <footer>
+        <section class="total">
+            <small>Total</small>
+            <p>$ 29.96</p>
+        </section>
+        <section class="checkout">
```
Comment (yaqootturman) [inline yourOrders/order.html:94]: the design need to be more similar with design

**Review by yaqootturman — CHANGES_REQUESTED** [review]
Good job Abdullah, I left some comments to de

**Review by yaqootturman — APPROVED** [review]

**Review by Seham96 — APPROVED** [review]

**Review by mahmoudhammad309 — APPROVED** [review]
good work

**Pull request merged**